### DISCUSSION TITLE
TH not sized per TD after adding the inline style.

### DISCUSSION THREAD

**Discussion by markddrake** [twbs/bootstrap#37707] [Q&A]
I have the following code for the table

```
  	    <table id="serviceTable" class="table">
		  <thead>
		    <tr>
		      <th scope="col">ID</th>
		      <th scope="col">Duration (时间)</th>
		      <th scope="col">Description (描述)</th>
		      <th scope="col">Cash Price (现金价)</th>
		      <th scope="col">Card Price (卡片价格)</th>
		    </tr>
		  </thead>
		  <tbody id="serviceList"/>
        </table>
```
The tbody populated using a react component.

All works well, the table body populated as expected with the rows returned by the fetch() method inside the react component
All my th and td entries are aligned perfectly. 

However if the table is large i want to make the rows scrollable.

I added the following in-line css

```
	<style>
    #serviceTable {
      height: 28em;
    }
	#serviceList {
      overflow-y: auto;
      height: 27em;
    }
    thead,tbody {
      display:inline-block;
    }
	</style>
```

I almost got what I was looking for. If the table contains more than 8 rows I get a scrollbar that allows me to scroll through the rows. However for some reason the th contents are no longer aligned with the td contents.

**Comment by mdo** [comment]
That's because you changed the `display` of the `thread` and `tbody`. You'll have to find another way to modify the table.
